src/components/ClickUIProvider/ClickUIProvider.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  Provider as TooltipProvider,
+  TooltipProviderProps,
+} from "@radix-ui/react-tooltip";
+import { ToastProps } from "@radix-ui/react-toast";
+import { ToastProvider } from "@/components/Toast/Toast";
+import { ThemeName } from "@/theme";
+import { ThemeProvider } from "@/theme/theme";
+
+type Props = TooltipProviderProps &
+  ToastProps & {
+    theme: ThemeName;
+  };
+
+const ClickUIProvider = ({
+  children,
+  theme,
+  delayDuration,
```

Comment:
i agree we shouldn't expose individual props like this. What does `delayDuration` refer to? it could apply to the toast, to the tooltip as well as to other components.